src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportMakerService;
+import java.util.Map;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    private static final String HEADER = "fruit,quantity";
+    private static final String NEW_LINE = "\n";
```

Comment:
Don't forget, that for some OS new line is `"\r\n"`
Don't create a constant for it, instead use some method from the class `System`